mmv1/products/apigee/Environment.yaml

```diff
@@ -114,13 +114,13 @@ properties:
     description: |
       Display name of the environment.
     required: false
-    immutable: true
+    immutable: false
```

Comment:
I noticed that this resource has multiple fields with `immutable` set to true. Is this PR change intended to make these 2 fields updatable, or the entire resource updatable?

Reply:
This change makes only `DisplayName` and `Description`.

Although the following issue refers only to Description, DisplayName is also included as updatable:
https://github.com/hashicorp/terraform-provider-google/issues/19820